flexget/manager.py

```diff
@@ -916,7 +918,7 @@ def shutdown(self, finish_queue=True):
         fire_event('manager.shutdown_requested', self)
         self.task_queue.shutdown(finish_queue)
 
-    def _shutdown(self):
+    def _cleanup(self):
```

Comment:
This rename is a bit inconvenient as it fires event shutdown ? Seems fine otherwise ...